# Generated by Django 3.0.5 on 2020-04-18 17:34

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):
	dependencies = [
		('accounts', '0002_delete_moderator'),
	]

	operations = [
		migrations.AlterField(
			model_name='user',
			name='email',
			field=models.EmailField(max_length=100, verbose_name='آدرس ایمیل'),
		),
		migrations.AlterField(
			model_name='user',
			name='first_name',
			field=models.CharField(max_length=30, verbose_name='نام'),
		),
		migrations.AlterField(
			model_name='user',
			name='last_name',
			field=models.CharField(max_length=30, verbose_name='نام خانوادگی'),
		),
		migrations.AlterField(
			model_name='user',
			name='password',
			field=models.CharField(max_length=128, verbose_name='password'),
		),
		migrations.AlterField(
			model_name='user',
			name='username',
			field=models.CharField(error_messages={'unique': 'A user with that username already exists.'},
								   help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.',
								   max_length=150, unique=True,
								   validators=[django.contrib.auth.validators.UnicodeUsernameValidator()],
								   verbose_name='username'),
		),
	]
